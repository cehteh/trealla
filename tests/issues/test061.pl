:- initialization(main).

main :-
    maplist(col([[1,2],[3,4]]), [1,2], X),
    writeln(X),
    halt.

col(Matrix, N, Column) :-
    maplist(nth1(N), Matrix, Column).
