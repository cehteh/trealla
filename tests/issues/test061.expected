[[1,3],[2,4]]
[[A,3],[2,A]]
